### ISSUE TITLE
Servant of Yogg-Saron tryouts: Curious Glimmerroot may offer spells as the correct choice, although decks are filled with only minions

### ISSUE THREAD

**Issue by PattuX** [HearthSim/hs-bugs#732]
* https://www.reddit.com/r/hearthstone/comments/67pxmg/possible_curious_glimmerroot_bug_in_this_weeks/?utm_content=comments&utm_medium=hot&utm_source=reddit&utm_name=hearthstone

*https://www.reddit.com/r/hearthstone/comments/67rgig/so_i_failed_my_pick_on_curios_glimmerroot_in_this/

**Comment by troggnostupidhs** [comment]
> Almost but not quite. When you queue up for a "random cards" brawl, you queue with one of the preconstructed decks (the ones you have for each class when you start playing). The cards all get replaced at the very beginning of the game.
>
 >It's one of the more annoying hacks in the game, which causes a bunch of stats to be very hard to parse :(

**Comment by troggnostupidhs** [comment]
Is this issue still considered a bug?

**Comment by jleclanche** [comment]
The title of the issue doesn't match with the links. Has nothing to do with spells or minions.

**Comment by culinko** [comment]
I guess the title should be changed to "Servant of Yogg-Saron tryouts: Curious Glimmerroot may offer cards that are not in your deck as the correct choice" then? Also the description about you not having a deck at the start of the game is incorrect, since you have the "basic deck for your class" as correctly mentioned in the trogg's quote.

**Comment by troggnostupidhs** [comment]
Not sure if this is a separate issue. In the ShiftCon Tavern Brawl, Curious Glimmerroot's battlecry does not trigger.

https://us.battle.net/forums/en/hearthstone/topic/20758617053

https://www.reddit.com/r/hearthstone/comments/6soex8/psa_curious_glimmerroot_totally_fizzles_in_this/

https://www.reddit.com/r/hearthstone/comments/6soha0/curious_glimmerroot_no_battlecry_one_more_bug/

https://www.reddit.com/r/hearthstone/comments/6sps7j/curious_glimmerroot_bug_in_tavern_brawl/

https://www.reddit.com/r/hearthstone/comments/6sp2xe/curious_glimmerroot_doesnt_work_in_the_current/

https://www.reddit.com/r/hearthstone/comments/6sqxtc/curious_glimmerroot_doesnt_work_in_the_brawl/

https://www.reddit.com/r/hearthstone/comments/6sqvqf/possible_bug_curious_glimmerroot_created_by/

https://www.reddit.com/r/hearthstone/comments/6ssf3e/curious_glimmerroot_doesnt_do_anything_in_tavern/

**Comment by troggnostupidhs** [comment]
Also not triggering in Encounter at the Crossroads

https://www.reddit.com/r/hearthstone/comments/72uxhl/why_doesnt_curious_glimmerroot_in_this_weeks/

https://www.reddit.com/r/hearthstone/comments/72vne6/i_just_played_curious_glimmerroot_and_it/

**Comment by troggnostupidhs** [comment]
https://www.reddit.com/r/hearthstone/comments/7mhf9l/curious_glimmerroot_doesnt_activate_in_tavern/

**Comment by troggnostupidhs** [comment]
This bug is likely 'Randomly constructed Tavern Brawl decks start as preconstructed decks'. It's also causing problems with Leyline Manipulator.

https://www.reddit.com/r/hearthstone/comments/7mmkf6/kinda_this_is_a_buggame_starts_with_30_reaven/

![image](https://user-images.githubusercontent.com/22508641/34414663-8598dde4-ebb8-11e7-8fa7-bcebf846a51f.png)

**Comment by troggnostupidhs** [comment]
https://youtu.be/C_iWCv9h9FY?t=165

**Comment by troggnostupidhs** [comment]
https://www.reddit.com/r/hearthstone/comments/aj5fhi/tavern_brawl_this_week_is_shiftcon_west_jan_23/eetba4f/

ShiftCon West has the same issue as ShiftCon where Glimmerroot does not trigger

**Comment by troggnostupidhs** [comment]
Deck cards are being replaced in this weeks Tavern Brawl, Dr. Boom's Ignoblegarden

It causes these issues:

https://us.forums.blizzard.com/en/hearthstone/t/talanji-brawl-bug/3895

> Playing Princess Talanji in the brawl pulls the bombs out of your hand.

https://us.forums.blizzard.com/en/hearthstone/t/the-mistcaller-bug-in-the-new-brawl/3891

> 
> The Mistcaller is not working properly. Apparently it doesn’t buff the “brawl only” minions in your deck.
> https://ibb.co/GT2nvfs
> https://ibb.co/G0LBMfZ

**Comment by troggnostupidhs** [comment]
https://www.reddit.com/r/hearthstone/comments/bgx83i/all_you_have_to_do_is_choose_the_card_that/

![image](https://user-images.githubusercontent.com/22508641/56697465-a636ba00-66bc-11e9-9898-8cd14de6725d.png)

The card from the Tavern Brawl set being showed indicates that Curious Glimmerroot is behaving better in this brawl.

This weeks brawl:
https://www.reddit.com/r/hearthstone/comments/bgwdja/this_weeks_tavern_brawl_is_hall_of_champions/

**Comment by Jetz72** [comment]
Still happens in some cases.
https://www.reddit.com/r/hearthstone/comments/e2l2ic/bug_curious_glimmerroot_does_not_show_3_cards_in/
>![](https://preview.redd.it/8b45fj3o3a141.jpg?width=1024&auto=webp&s=558a034a6731bf32e31f8f51ebef488c09ddae68)

Brawl is Encounter at the Crossroads. Perhaps they haven't bothered to fix it for old brawls.
